Realm/RLMRealm.h

```diff
@@ -815,6 +815,16 @@ NS_REFINED_FOR_SWIFT;
  */
 - (struct RLMClassPrivileges)privilegesForClassNamed:(NSString *)className;
 
+#pragma mark - Utilities
+
+/**
+ Returns the number of active versions of this Realm.
+
+ There will always be at least 2 active versions of the Realm
+ because the previous version won't be cleaned up until the next commit.
+*/
```

Comment:
Maybe something like `@see RLMRealmConfiguration.maximumNumberOfActiveVersions` since that tries to describe what active versions mean. I'm not sure that that's the correct syntax.